monolithic-architecture/README.md

```diff
@@ -0,0 +1,136 @@
+## Monolithic-Ecommerce App
```

Comment:
`README.md` has a very specific format that is common for all the patterns. See instructions at: https://github.com/iluwatar/java-design-patterns/wiki/01.-How-to-contribute

Specifically, take a look at the required front matter structure at the beginning of the file.